packages/lesswrong/components/community/Community.tsx

```diff
@@ -110,6 +119,16 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
     display: 'inline-block',
     marginLeft: 6
   },
+  inactiveGroups: {
+    '& .MuiIconButton-label': {
+      color: theme.palette.grey[600],
+    },
+    '& .MuiFormControlLabel-label': {
```

Comment:
[nit] I don't think that's the MUI-idiomatic way to address MUI subcomponents, but I'll allow it.